README.md

```diff
@@ -84,6 +86,34 @@ Features comprise the following elements:
 
 See the available Busola [feature flags](docs/features.md) for more information.
 
+#### Environment specific settings
+
+You can provide override to the default configuration with your own environment specific settings.
+Custom environment directory structure should look and be placed in **public/environments**.
+
+```
+custom-env/
+├── config
+│   └── config.yaml
+└── extensions
+    ├── extensions.yaml
+    └── wizards.yaml
+```
+
+> **Important:** Files `extensions.yaml`, `statics.yaml`, `wizards.yaml` and `config.yaml` are required to make busola work properly.
+
+To activate environment configuration you have to create or edit `active.env` file located in [public](./public).
+The `active.env` file should look like that:
+
+```dotenv
+ENVIRONMENT=your-environment-name
+```
+
+When the `ENVIRONMENT` is set to `my-env`, the busola looks for custom configuration in **public/environemnt/my-env**
+If the `ENVIRONMENT` is not set busola fetch default configuration which has the same structure as custom configuration located in [public](./public)
+
+In case of docker image, the file `active.env` is created from `ENVIRONMENT` env at the startup to image.
```

Comment:
```suggestion
In the case of the Docker image, the `active.env` file is created from the **ENVIRONMENT** env at the startup of the image.
```

Reply:
Again, I'm not sure if I understand it correctly. What is "env" here? If it's environment, is it necessary to say "ENVIRONMENT environment"?

Reply:
env is a shortcut of environment variable which is represented by `KEY=value`, so to make it more readable I used `env`. Additionaly it's quite common to name it like that.

Reply:
If it's not the actual value, I recommend rephrasing it to:
In the case of the Docker image, the `active.env` file is created at the startup of the image from the environment specified in the **ENVIRONMENT** variable.